src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v3/ActionListenerV2.java

```diff
@@ -0,0 +1,15 @@
+package org.sagebionetworks.web.client.widget.entity.menu.v3;
+
+import org.sagebionetworks.web.client.jsinterop.ReactMouseEvent;
+import org.sagebionetworks.web.client.widget.entity.menu.v2.Action;
+
+@FunctionalInterface
+public interface ActionListenerV2 {
```

Comment:
"ActionListenerV2" in v3 of action menu.  Should this be renamed to ActionListenerV3?

Reply:
Will rename (and probably drop the 'V#' altogether) after removing old code